image/descriptor.go

```diff
@@ -32,6 +33,10 @@ type descriptor struct {
 	Size      int64  `json:"size"`
 }
 
+func (d *descriptor) normalizeDigest() string {
+	return strings.Replace(d.Digest, ":", "-", -1)
+}
```

Comment:
> I'm not sure what's holding #159 up, it's been out there for a while
> ;).  So I'm not sure what timeframe to expect before people can use it
> (although folks are certainly free to use it now, without waiting on
> the branch to land in this repo ;).

I don't know either - just that you PR is 1000+ loc changed - while this one is just pulling in 10 additions (https://github.com/opencontainers/image-spec/pull/177/commits/0e8f74b634b84f56e6fcb46b68971219fd3d9e55) to fix a fatal bug.

I believe this should go in asap. Potentially #159 can take a lot more time to be reviewed (not maintainers reviewed it yet)